argo-cd/kueyen-values.yaml

```diff
@@ -0,0 +1,32 @@
+redis:
```

Comment:
Is kueyen the name of the environment?  This is a word I've never seen

Reply:
Yes.  I don't know what it means either.